style.css

```diff
@@ -0,0 +1,430 @@
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+body {
+  width: 375px;
+}
+
+/* header part css */
+
+header {
+  background: #3c3a39;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navbar img {
+  margin: 19px 0 12px 20px;
+  cursor: pointer;
+}
+
+.navbar .toggler {
+  margin: 19px 38px 12px 0;
+}
+
+/* main part css */
+
+.main {
+  background: #1c1a19;
+}
+
+article {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.main-article h2 {
+  padding-top: 120px;
+  width: 80%;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.main-article h3 {
+  padding-top: 12px;
+  width: 80%;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: #fff5e1;
+}
+
+.main-article p {
+  padding-top: 12px;
+  width: 80%;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.social-media {
+  position: relative;
+}
+
+.social-media-icons {
+  display: flex;
+  justify-content: flex-start;
+  margin: 12px 0 130px 26px;
+  padding-bottom: 130px;
+}
+
+.social-media-icons li {
+  margin-left: 22.92px;
+}
+
+.social-media .main-img {
+  position: absolute;
+  top: -120px;
+  right: 0;
+}
+
+/* work section start */
+
+.work {
+  margin: 0 auto;
```

Comment:
I suggest you use `display:grid` for the work section as instructed in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)
<img width="1248" alt="Screenshot 2023-02-18 at 08 34 53" src="https://user-images.githubusercontent.com/48546992/219843131-eae6f591-c390-4b2a-b87c-36cc9ac5183a.png">